src/modules/rewards/Rewards.tsx

```diff
@@ -83,6 +84,7 @@ const Rewards: FC<RewardsProps> = () => {
         />
 
         {activeTab === 'dashboard' && <ReferralSection handleUnlockProfile={handleUnlockProfile} />}
+        {activeTab === 'activity' && <RewardsActivitiesBottomSection />}
```

Comment:
Any particular reason for doing this here. Since this is a part of rewards page. I think this should be inside tabs component similar to the earlier activities.

Reply:
![Screenshot 2024-08-05 at 15 00 08](https://github.com/user-attachments/assets/fd67ac01-d399-48eb-accb-3e31be71ee9e)


The Boxes will appear all chunked as one if i put then within the rewards tab container unless i restructure the whole thing. RewardsActivitiesBottomSection will be outside of the container so we can the desired spacing btw each components. But i can improve on the structure accross both components